README.md

```diff
@@ -77,12 +77,14 @@ Or using a plain text file stored on the filesystem:
 ```puppet
 file { '/tmp/bar_set_content':
   ensure  => present,
-  content => "1.2.3.0/24\n5.6.7.8/32"
+  content => "1.2.3.0/24\n5.6.7.8/32",
 }
--> ipset { 'bar':
-  ensure => present,
-  set    => 'file:///tmp/bar_set_content',
-  type   => 'hash:net',
+
+ipset { 'bar':
+  ensure    => present,
+  set       => 'file:///tmp/bar_set_content',
+  type      => 'hash:net',
+  subscribe => File['/tmp/bar_set_content'],
```

Comment:
This changes it from a `require` to `subscribe`. Is that intentional?

Reply:
Personally, I prefer `->` over `subscribe/require`.

Reply:
Style guide (which we should either follow or submit PRs against) prefers the metaparameters. https://puppet.com/docs/puppet/latest/style_guide.html#chaining-arrow-syntax

Reply:
Just changing arrows to words, which is the preferred way to handle relationships.

Reply:
`->` would translate to `require`. `~>` would be `subscribe`.

Reply:
https://puppet.com/docs/puppet/latest/lang_relationships.html#refreshing-and-notification

Reply:
I get it was a require, but it should be a notify. If the networks change, then the ipset should change, too.